charts/ccsm-helm/test/zeebe/golden/statefulset.golden.yaml

```diff
@@ -139,6 +139,16 @@ spec:
           defaultMode: 0744
       - name: exporters
         emptyDir: {}
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
I used here the `requiredDuringSchedulingIgnoredDuringExecution` since we have similar setting in our SaaS. This seems to be a hard requirement vs. the soft which would be `preferredDuringSchedulingIgnoredDuringExecution`

https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity